react/Gallery.js

```diff
@@ -54,8 +59,8 @@ Gallery.propTypes = {
   products: PropTypes.arrayOf(productShape),
   /** ProductSummary props. */
   summary: PropTypes.any,
-  /** Layout mode of the gallery */
-  layoutMode: PropTypes.string,
+  /** Layout mode of the gallery in mobile view */
+  mobileLayoutMode: PropTypes.string,
```

Comment:
Are the layout modes discrete? You could use PropTypes' `oneOf' to make it more clear. Maybe add some constantes on the top of the page and use it in the conditionals...

Reply:
There are some proptypes that are not required and algo they are not on defaultProps. Pls check

Reply:
Fixed.